<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="title_label">
     <property name="text">
      <string>PMPS SC DB Management Tool</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="plc_label">
         <property name="text">
          <string>PMPS-enabled PLCs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="plc_table"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="device_label">
         <property name="text">
          <string>Configured Devices in PLC DB</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="device_list"/>
       </item>
       <item>
        <widget class="QLabel" name="param_label">
         <property name="text">
          <string>Specific Device Parameters in PLC DB</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="param_table"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
